src/main/java/net/dv8tion/jda/api/entities/User.java

```diff
@@ -220,6 +223,50 @@ default String getEffectiveAvatarUrl()
         return avatarUrl == null ? getDefaultAvatarUrl() : avatarUrl;
     }
 
+    /**
+     * The Discord Id for this user's banner image.
+     * If the user has not set a banner, this will return null.
+     * If the user has not been retrieved using {@link JDA#retrieveUserById(long)}, this will return null.
+     *
+     * @throws UnsupportedOperationException
+     *         If this User was created with {@link #fromId(long)}
+     *
+     * @return Possibly-null String containing the {@link net.dv8tion.jda.api.entities.User User} banner id.
+     */
+    @Nullable
+    String getBannerId();
+
+    /**
+     * The URL for the user's banner image.
+     * If the user has not set a banner, this will return null.
+     * If the user has not been retrieved using {@link JDA#retrieveUserById(long)}, this will return null.
+     *
+     * @throws UnsupportedOperationException
+     *         If this User was created with {@link #fromId(long)}
+     *
+     * @return Possibly-null String containing the {@link net.dv8tion.jda.api.entities.User User} banner url.
+     */
+    @Nullable
+    default String getBannerUrl()
+    {
+        String bannerId = getBannerId();
+        return bannerId == null ? null : String.format(BANNER_URL, getId(), bannerId, bannerId.startsWith("a_") ? "gif" : "png");
+    }
+
+    /**
+     * The user's accent color.
+     * If the user has not set an accent color, this will return null. The automatically calculated color is not returned.
+     * If the user has set a banner, this accent color will not be shown.
+     * If the user has not been retrieved using {@link JDA#retrieveUserById(long)}, this will return null.
+     *
+     * @throws UnsupportedOperationException
+     *         If this User was created with {@link #fromId(long)}
+     *
+     * @return Possibly-null {@link java.awt.Color} containing the {@link net.dv8tion.jda.api.entities.User User} accent color.
+     */
+    @Nullable
+    Color getAccentColor();
```

Comment:
Add `int getAccentColorRaw()` similar to `Role#getColorRaw`.

https://github.com/DV8FromTheWorld/JDA/blob/969b04ff777e8a97b36c7af3a9e71bc2918243ab/src/main/java/net/dv8tion/jda/api/entities/Role.java#L119-L125